File: 5543/5543.py
p1 = 2000
p2 = 2000
for i in range(0, 3):
    tmp = int(input())
    if p1 > tmp:
        p1 = tmp
for i in range(0, 2):
    tmp = int(input())
    if p2 > tmp:
        p2 = tmp
print(p1+p2-50)

File: 5543/5543.c
#include <stdio.h>
int main() {
	int p1 = 2000;
	int p2 = 2000;
	int tmp;
	for (int i = 0; i < 3; i++) {
		scanf("%d", &tmp);
		if (p1 > tmp)
			p1 = tmp;
	}
	for (int i = 0; i < 2; i++) {
		scanf("%d", &tmp);
		if (p2 > tmp)
			p2 = tmp;
	}
	printf("%d", p1 + p2 - 50);
	return 0;
}